Exiled.API/Features/Camera.cs

```diff
@@ -233,7 +233,7 @@ public bool IsBeingUsed
         /// </summary>
         /// <param name="camera079">The base <see cref="Scp079Camera"/>.</param>
         /// <returns>A <see cref="Camera"/> or <see langword="null"/> if not found.</returns>
-        public static Camera Get(Scp079Camera camera079) => Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
+        public static Camera Get(Scp079Camera camera079) => camera079 != null && Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
```

Comment:
Uh, it will try to create camera wrapper when base is null?

Reply:
skill issue